src/compiler/CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ if(ROCM_FOUND)
                 ${ROCM_LIBRARIES}
                 -L${ROCM_LIBRARIES_DIR}/../hsa/lib -L${ROCM_LIBRARIES_DIR}
                 -Wl,--enable-new-dtags
-                -Wl,-rpath,\$ORIGIN/../../hsa/lib -Wl,-rpath,\$ORIGIN/../../lib
+                -Wl,-rpath,\$ORIGIN/../hsa/lib -Wl,-rpath,\$ORIGIN/../lib
```

Comment:
The ATMI package included in ROCm release installs libatmi_runtime.so at /opt/rocm-dir/atmi/lib/ whereas aomp includes the same library under /opt/rocm/aomp/lib in the ROCm+aomp bundled release. The relative path to find libhsa-runtime64.so.1 is $ORIGIN/../../hsa/lib, and the relative path to find libamd_comgr.so.1 would be either $ORIGIN by itself to pick up libamd_comgr.so.1 from opt/rocm/aomp/lib or $ORIGIN/../../lib to pick libamd_comgr.so.1 from opt/rocm/lib that gets installed as part of ROCm release of comgr package.

It looks to me that the above paths would need to be adjusted, as $ORIGIN/../hsa/lib would become opt/rocm/atmi/lib/../hsa/lib which will not be the correct location of libhsa-runtime64.so.1, and similarly $ORIGN/../lib would be expanded to opt/rocm/atmi/lib/../lib which would be the same as $ORIGIN, and that should be ok to pick libamd_comgr.so.1 from aomp lib directory but won't point to ROCm installation of comgr package location /opt/rocm/lib.